src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/idam/IdamTestService.java

```diff
@@ -58,37 +69,66 @@ public IdamTestService(
 
     public User createSolicitor() {
         String email = testData.nextUserEmail();
-        createUser(createSolicitorRequest(email, aatConfiguration.getSmokeTestSolicitor().getPassword()));
-        return userService.authenticateUser(email, aatConfiguration.getSmokeTestSolicitor().getPassword());
+        return Failsafe.with(retryPolicy)
+            .get(() -> {
+                try {
+                    idamTestApi.createUser(
+                        createSolicitorRequest(email, aatConfiguration.getSmokeTestCitizen().getPassword())
+                    );
+                } catch (ForbiddenException ex) {
+                    ex.printStackTrace();
+                }
+                return userService.authenticateUser(email, aatConfiguration.getSmokeTestSolicitor().getPassword());
+            });
     }
 
     public User createCitizen() {
-        String email = testData.nextUserEmail();
-        createUser(createCitizenRequest(email, aatConfiguration.getSmokeTestCitizen().getPassword()));
-        return userService.authenticateUser(email, aatConfiguration.getSmokeTestCitizen().getPassword());
+        String email = "mryan321.hmcts.claimant@gmail.com"; //testData.nextUserEmail();
```

Comment:
Presumably that should be replaced with the commented code?

Reply:
O..M..G 👀